optaweb-vehicle-routing-backend/src/main/resources/application.properties

```diff
@@ -25,16 +27,9 @@ logging.level.org.optaplanner.core=info
 logging.level.org.optaweb.vehiclerouting=info
 
 # Datasource
-# - using an embedded DB with relative path:http://h2database.com/html/features.html#embedded_databases
-# - not closing the DB automatically: http://h2database.com/html/features.html#closing_a_database
-spring.datasource.url=jdbc:h2:file:./local/db/vrp;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
-spring.datasource.username=sa
-spring.datasource.password=
-spring.datasource.driver-class-name=org.h2.Driver
+spring.datasource.url=jdbc:postgresql://postgresql:5432/${DATABASE_NAME}
```

Comment:
@Christopher-Chianelli Here, `postgresql:5432` is the hard-coded name and port of the OpenShift Service created by `oc new-app postgresql-persistent`.

Reply:
What comment are you replying to? I can't seem to find it and I don't recall commenting on it. Are you referring to the `pom.xml` comment? I don't think you need to declare postgres in `pom.xml` to use it; my worry is it get packaged with the JAR for those who don't use it (but I don't think these JAR's are shaded, so they should be good?)

Reply:
This is in response to https://github.com/kiegroup/optaweb-vehicle-routing/pull/223#discussion_r344987620.

The postgresql dependency in POM is the JDBC driver. I believe that is needed.

Reply:
Ah, I thought the dependency was for the entire posgresql database; I will mark the comment as resolve then